### PR TITLE
[IMP] mail: Disable message, mail and notification duplication

### PR DESCRIPTION
**Pull request by reth-odoo**
Duplicating an email sent from Odoo would raise an error because some required fields could not be set. 
Setting those fields on duplication does not make sense, duplication is to be disabled on e-mails.

As this applies to some messages and no use-case could be found for the duplication of messages or notifications,
both also had duplication disabled.

Task-2857244
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/models/mail_mail.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from odoo import tools
 from odoo.addons.base.models.ir_mail_server import MailDeliveryException
 from odoo.tools.safe_eval import safe_eval
+from odoo.exceptions import UserError
```
Comment (std-odoo) [inline addons/mail/models/mail_mail.py:18]: same here
```
[IMP] mail: disable message, mail and notification duplication

```

File: addons/mail/models/mail_notification.py [hunk 2]
```diff
@@ -79,6 +79,10 @@ def write(self, vals):
             vals['read_date'] = fields.Datetime.now()
         return super(Notification, self).write(vals)
 
+    @api.returns('self', lambda value: value.id)
```
Comment (std-odoo) [inline addons/mail/models/mail_notification.py:82]: It's not necessary imo
PS: when you add a new translation in stable, you should export the .po file (in the technical view)

**Review by std-odoo — COMMENTED** [review]
2 small comments and we should be good :)

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
@fw-bot ignore

**Pull request closed without merging**